pkg/resource/deploy/step_generator.go

```diff
@@ -769,6 +769,19 @@ func (sg *stepGenerator) GenerateDeletes(targetsOpt map[resource.URN]bool) ([]St
 	return dels, nil
 }
 
+func (sg *stepGenerator) getTargetIncludingChildren(target resource.URN) map[resource.URN]bool {
+	allTargets := make(map[resource.URN]bool)
+	allTargets[target] = true
+
+	for _, res := range sg.deployment.prev.Resources {
```

Comment:
Probably worth adding a note that this approach only works b/c `sg.deployment.prev.Resources` is topologically-sorted